packages/nx-openapi/package.json

```diff
@@ -10,11 +10,10 @@
     "directory": "packages/nx-openapi"
   },
   "dependencies": {
-    "openapi-typescript": "^7.5.2",
-    "enquirer": "^2.3.6",
-    "@nx/devkit": "19.8.3",
-    "@redocly/openapi-core": "^1.27.2",
-    "vitest": "2.0.0"
```

Comment:
Just triple-checking that vitest being removed is all good and intentional?

Reply:
Yes. It's intended. We should not have vitest as a dependency. It's a dev dependency.